### ISSUE TITLE
Error when prepare

### ISSUE THREAD

**Issue by AchmadFathoni** [bartoszek/AUR-cloudcompare#11]
```
Checking patch Makefile...
error: while searching for:
# use the second line to disable profiling instrumentation
# PROFILING := -pg
PROFILING :=
CCFLAGS   := -Wall $(INC) $(CONFIG) -O2 -DNDEBUG $(PROFILING)
CXXFLAGS  := $(CCFLAGS) $(CPP11_FLAGS)
CCDFLAGS  := -Wall $(INC) $(CONFIG) -ggdb
CXXDFLAGS := $(CCDFLAGS)

error: patch failed: Makefile:62
error: Makefile: patch does not apply
Checking patch src/mesh/mesh.isct.tpp...
error: while searching for:
        // DGM: to replace lambda in IsctProblem constructor!
        void quantizeVerts(const Quantization& quantizer)
        {
                if (!mesh)
                        return;

                size_t N = mesh->verts.size();
                quantized_coords.resize(N);

                uint write = 0;
                for (Vptr v = verts.getFirst(); v != NULL; v = verts.getNext(v))
                {
#ifdef _WIN32
                        Vec3d raw = mesh->verts[v->ref].pos;

error: patch failed: src/mesh/mesh.isct.tpp:650
error: src/mesh/mesh.isct.tpp: patch does not apply
Checking patch src/util/prelude.h...
error: while searching for:
        err()     << "ENSURE FAILED at " \
                  << __FILE__ << ", line #" << __LINE__ << ":\n" \
                  << "    " << #STATEMENT << std::endl; \
                throw std::exception("ENSURE FAILED"); \
    } \
}
#endif // ENSURE

```

**Comment by AchmadFathoni** [comment]
It seem that `cork.patch` is already merged.

**Comment by bartoszek** [comment]
Fixed in e3fd2672846d2b11d17febbe15bec1648ac14b29
